Downloads/WeatherApplication/app/src/main/java/com/example/weatherapplication/ForecastActivity.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.weatherapplication
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.example.weatherapplication.adapter.ForecastAdapter
+import com.example.weatherapplication.databinding.ActivityForecastBinding
+import com.example.weatherapplication.model.Forecast
+import com.example.weatherapplication.model.ForecastTemp
+import java.util.ArrayList
+
+class ForecastActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityForecastBinding
+    private var forecastList = ArrayList<Forecast>()
```

Comment:
You should be using Kotlin collection classes, not Java ones. In the future use `listOf` Kotlin function to create lists.